accepted/future-releases/extension-types/feature-specification.md

```diff
@@ -976,24 +976,18 @@ declaration itself, or we're talking about a particular generic
 instantiation of an extension type. *For non-generic extension type
 declarations, the representation type is the same in either case.*
 
-Let `V` be an extension type of the form
-<code>Name\<T<sub>1</sub>, .. T<sub>s</sub>&gt;</code>, and let
-`R` be the corresponding instantiated representation type.  If `R` is
-non-nullable then `V` is a proper subtype of `Object`, and `V` is
-non-nullable.  Otherwise, `V` is a proper subtype of `Object?`, and
-`V` is potentially nullable.
+An extension type `V` is a proper subtype of `Object?`. It is potentially
+non-nullable, unless it implements `Object` or a subtype thereof
+*(as described in the section about extension types with superinterfaces)*.
 
 *That is, an expression of an extension type can be assigned to a top type
-(like all other expressions), and if the representation type is
-non-nullable then it can also be assigned to `Object`. Non-extension types
-(except bottom types and `dynamic`) cannot be assigned to extension types
-without an explicit cast. Similarly, null cannot be assigned to an
-extension type without an explicit cast, even in the case where the
-representation type is nullable (even better: don't use a cast, call a
-constructor instead). Another consequence of the fact that the extension
-type is potentially non-nullable is that it is an error to have an instance
-variable whose type is an extension type, and then relying on implicit
-initialization to null.*
+(like all other expressions). Non-extension types (except bottom types and
+`dynamic`) cannot be assigned to extension types without an explicit cast.
+Similarly, null cannot be assigned to an extension type without an explicit
+cast (even better: don't use a cast, call a constructor instead). Another
+consequence of the fact that the extension type is potentially non-nullable is
+that it is an error to have an instance variable whose type is an extension
```

Comment:
Long sentence.

Consider changing 
> Another consequence of the fact that the extension type is potentially non-nullable is that it ....
to
> Because an extensiont type is potentially nullable, it

Or maybe even continue:

> Because an extension type is potentially non-nullable, 
> an instance variable whose type is an extension type must be initialized.
> It will not be implicitly initialized to `null`.

You can't "rely on implicit initialization to null", when implicit initializtion to null doesn't happen. You can try, but that's not what the code means.
It's better to say what is, than what could have been, had things been different, in some unspecified way (and not another unspecified way).